packages/strapi-parts/src/Box/Box.stories.mdx

```diff
@@ -20,7 +21,7 @@ import { Box } from '@strapi/parts/Box';
 
 ## Usage
 
-Description...
+Default Box component.
```

Comment:
The Box component is a very powerful component and it is used almost everywhere in the app. IMO we should give more details on that component. 

What do you thing about something like that:
```js
The Box component will allow you to create your containers according to the guidelines of the Strapi design system (border-radius, shadows, background-color, padding, border, etc).
```